### PR TITLE
Evaluation

### PR DESCRIPTION
**Pull request by MJaschewski**

### CODE REVIEW

File: backend/src/main/java/de/neuefische/backend/ReactRoutingForwarding.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+public class ReactRoutingForwarding implements WebMvcConfigurer {
+
+    public static final String DEFAULT_STARTING_PAGE = "static/index.html";
+
+    static class ReactRoutingPathResourceResolver extends PathResourceResolver {
+        @Override
+        protected Resource getResource(String resourcePath, Resource location) throws IOException {
+            var requestedResource = location.createRelative(resourcePath);
+
+            // Is this a request to a real file?
```
Comment (SvenjaKlenke) [inline backend/src/main/java/de/neuefische/backend/ReactRoutingForwarding.java:22]: Please remove comment

File: backend/src/main/java/de/neuefische/backend/ReactRoutingForwarding.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            if (requestedResource.exists() && requestedResource.isReadable()) {
+                return requestedResource;
+            }
+
+            // It seems to be only a frontend-routing request (Single-Page-Application).
```
Comment (SvenjaKlenke) [inline backend/src/main/java/de/neuefische/backend/ReactRoutingForwarding.java:27]: Please remove comment

File: backend/src/test/java/de/neuefische/backend/ReactRoutingPathResourceResolverTest.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        var resolver = new ReactRoutingPathResourceResolver();
+        var location = mock(Resource.class);
+        var relativeLocation = mock(Resource.class);
+        var resourcePath = "index.html";
+        when(location.createRelative(resourcePath)).thenReturn(relativeLocation);
+        when(relativeLocation.exists()).thenReturn(true);
+        when(relativeLocation.isReadable()).thenReturn(true);
+
+        // WHEN
+        var actual = resolver.getResource(resourcePath, location);
```
Comment (SvenjaKlenke) [inline backend/src/test/java/de/neuefische/backend/ReactRoutingPathResourceResolverTest.java:28]: Why do you use "var"?
It look like a frontend test, we don´t understand this

**Review by SvenjaKlenke — APPROVED** [review]
I don't understand the most things, but it looks good.

File: backend/src/test/java/de/neuefische/backend/ReactRoutingPathResourceResolverTest.java (hunk 3 above)
Comment (MJaschewski) [reply backend/src/test/java/de/neuefische/backend/ReactRoutingPathResourceResolverTest.java:28]: This was given from the coaches to fix the white label errors when delployed.

**Pull request merged**